num/percentage.go

```diff
@@ -80,6 +80,11 @@ func (p Percentage) StringWithoutSymbol() string {
 	return v.String()
 }
 
+// Rescale will rescale the percentage value to the provided exponent.
+func (p Percentage) Rescale(exp uint32) Percentage {
+	return Percentage{Amount: p.Amount.Rescale(exp + 2)}
```

Comment:
I think the `+ 2` bit might be a bit confusing. You can see the conflict in the tests:
```go
p := num.MakePercentage(160, 3)
p.String() // 16.0%
p = p.Rescale(2)
p.String() // I'd expect 16%
p = p.Rescale(4)
p.String() // I'd expect 16.00%
```

Reply:
Fixed 👍🏼